releases/release-1.10/release-notes-draft.md

```diff
@@ -1,3 +1,997 @@
-# Release 1.10
+# Kubernetes 1.10 Release Notes Draft
 
-This will will be used to track release notes for Kubernetes 1.10.
+# Major Themes
+
+### Node
+
+Many of the changes within SIG-Node revolve around control, with the beta release of [Dynamic Kubelet Configuration](https://github.com/kubernetes/features/issues/281), the ability to make changes to Kubelet without having to bring down the node, and alpha support for the ability to [configure whether containers in a pod should share a single process namespace](https://github.com/kubernetes/features/issues/495), The CRI has also seen some improvements, with support for  [support for Windows Container Configuration](https://github.com/kubernetes/features/issues/547) and the beta release of the [CRI validation test suite](https://github.com/kubernetes/features/issues/292). In addition, this release includes alpha support for [Debug Containers](https://github.com/kubernetes/features/issues/277).
```

Comment:
s/The CRI has also seen some improvements/The CRI has also beem upgrade to [v1alpha2](https://github.com/kubernetes/kubernetes/tree/master/pkg/kubelet/apis/cri/runtime/v1alpha2)/

Reply:
Done.